lms/djangoapps/commerce/api/v0/views.py

```diff
@@ -84,6 +84,7 @@ def post(self, request, *args, **kwargs):
         """
         Attempt to enroll the user.
         """
+        return DetailResponse('Method not allowed', status=HTTP_406_NOT_ACCEPTABLE)
```

Comment:
The code after `return` is useless, do we need to keep it?

Reply:
Yep, good point! :-) I guess, we don't need the rest for now.